Marlin/src/module/motion.cpp

```diff
@@ -621,7 +621,7 @@ void restore_feedrate_and_scaling() {
       ) {
         const float dist_2 = HYPOT2(target.x - offs.x, target.y - offs.y);
         if (dist_2 > delta_max_radius_2)
-          target *= delta_max_radius / SQRT(dist_2); // 200 / 300 = 0.66
+          target *= (float)(delta_max_radius / (SQRT(dist_2))); // 200 / 300 = 0.66
       }
```

Comment:
Please count the '(' and the ')'.

Reply:
GCC will optimize them out, was just being pedantic, want me to fix it?

Reply:
Sorry. Was confused by the highlighting.
Now, with the glases on, the counts match.

Reply:
No problem, happens to everyone. :-)